api/routes/qaris.js

```diff
@@ -42,4 +44,68 @@ router.get('/related/:id', (req, res) => {
     .then(related => res.send(related));
 });
 
+router.get('/:id/download', (req, res) => {
+  const archive = archiver('zip', {
+    zlib: { level: 9 } // Sets the compression level.
+  });
+  const header = {
```

Comment:
this is unlikely needed inside of this request? It could be at the top of this file or in another constants file?

Reply:
Nice catch, top of the file maybe? as it's used only with the file?